files/src/hobby_hub_utils/test/test_pin.py

```diff
@@ -0,0 +1,20 @@
+from hobby_hub_pkg.pin_manager import get_led
+import unittest
+import time
+
+class TestPinManager(unittest.TestCase):
```

Comment:
also a test where you request and use a pin in one file then request the same pin in another to view the state would be a useful test.  

from my perspective, it should be able to view the state in the second program as it changes in the first.

Reply:
Sure, although I don't see why they have to be from separate files? If you request a pin and then request the same pin in the same file, shouldn't the behavior be the same?

Reply:
well i guess the idea is that you have a common pin between two programs. so having two files kinda simulates that.